packages/scenes-app/src/demos/scopesDemo.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  AdHocFiltersVariable,
+  SceneAppPage,
+  SceneAppPageState,
+  SceneFlexItem,
+  SceneFlexLayout,
+  SceneVariableSet,
+  ScopesVariable,
+  VizPanel,
+} from '@grafana/scenes';
+import { getEmbeddedSceneDefaults, getPromQueryInstant } from './utils';
+import { EmbeddedSceneWithContext } from '@grafana/scenes-react';
+
+export function getScopesDemo(defaults: SceneAppPageState) {
+  return new SceneAppPage({
+    ...defaults,
+    $variables: new SceneVariableSet({
+      variables: [new ScopesVariable({ enable: true }), new AdHocFiltersVariable({})],
```

Comment:
```suggestion
      variables: [new ScopesVariable({ enable: true }), new AdHocFiltersVariable({ layout: 'combobox' })],
```
This is needed to see scope injected filters